build/scripts/buildWebKitFromWithinXcode.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-xcodebuild -target JavaScriptCore -sdk "$SDKROOT" -configuration $CONFIGURATION ARCHS="$ARCHS" ONLY_ACTIVE_ARCH=$ONLY_ACTIVE_ARCH $DEPLOYMENT_TARGET_SETTING_NAME=${!DEPLOYMENT_TARGET_CLANG_ENV_NAME}
\ No newline at end of file
+xcodebuild -target JavaScriptCore -sdk "$SDKROOT" -configuration $CONFIGURATION ARCHS="$ARCHS" ONLY_ACTIVE_ARCH=$ONLY_ACTIVE_ARCH $DEPLOYMENT_TARGET_SETTING_NAME=${!DEPLOYMENT_TARGET_CLANG_ENV_NAME} GCC_WARN_INHIBIT_ALL_WARNINGS=YES
```

Comment:
@fealebenpae Does it make sense to add `xcpretty` here, too? This would reduce the build log in Xcode, that quite often makes it unresponsive.